scripts/genetic_backtester/darwin.js

```diff
@@ -240,6 +241,23 @@ let RangeNeuralActivation = () => {
 };
 
 let strategies = {
+  bollinger: {
+    period_length: RangePeriod(1, 60, 'm'),
+    min_periods: Range(1, 20),
```

Comment:
@station384 What about the min_periods? It's default value is 52. Shouldn't it range between 1-120 or 1-200 like the other strategies?